tipg/db.py

```diff
@@ -48,7 +54,7 @@ async def con_init(conn):
             await conn.execute(sqlfile.read_text())
 
     # Register TiPG functions
-    dbcatalogsql = pathlib.Path(__package__) / "sql" / "dbcatalog.sql"
+    dbcatalogsql = resources_files(__package__) / "sql" / "dbcatalog.sql"
```

Comment:
When `pathlib.Path(__package__)` is returning a relative path, when using tipg from another application it will then error not finding the sql files.